src/main/java/Duke.java

```diff
@@ -1,10 +1,51 @@
+import java.io.*;
```

Comment:
Perhaps it would be better to use explicit import statements. It can serve as a form of documentation regarding the dependencies a class has